src/core/src/pass/serialize.cpp

```diff
@@ -1224,21 +1224,34 @@ void ngfunction_2_ir(pugi::xml_node& netXml,
     }
 }
 
+const std::filesystem::path check_path_safety(const std::filesystem::path& path) {
```

Comment:
Re-use update current utils functions:
Use `ov::util::sanitize_path` which should remove traversal part from path.

There are also utils like in std like `canonical`, `weakly_canonical`